services/minespace-web/src/components/Forms/projects/informationRequirementsTable/IRTDownloadTemplate.js

```diff
@@ -19,7 +19,7 @@ export class IRTDownloadTemplate extends Component {
     return (
       <Row>
         <Col>
-          <Typography.Title level={4}>Download IRT template</Typography.Title>
+          <Typography.Title level={3}>Download IRT template</Typography.Title>
           <Typography.Paragraph>
             The first step in composing an IRT is to download the official{" "}
```

Comment:
I think there is a missing word at the end of this sentence: "...download the official...", I imagine it's probably template.

Reply:
it is a link containing "IRT template" in line 32